### PR TITLE
feat(trading): add resources dropdown to console nav

### LINKED ISSUE #2804: Feedback, help, docs - signposting and links
Blocked by #2840 

## Chore

There are several links we need to add to console to help tie up different user journeys 

* Docs 
* Provide feedback 
* [Link to mainnet incident reports#2764](https://github.com/vegaprotocol/frontend-monorepo/issues/2764)

This chore is to find / design a place to hold these, and implement. 

## Acceptance Criteria

- [ ] I can get to the docs site
- [ ] I can get to the feedback site

## Tasks

- [x] Explore and sketch
- [x] Team and stakeholder review
- [x] Visual Design
- [x] Dev
- [ ] Test

## Additional details / background info / ideas

DYDX 

Image

GMX

Image

Perp

Image
Image

GTrade

Image

OKX 

Image

Binance

Image

### PR DESCRIPTION
**Pull request by mattrussell36**
# Related issues 🔗

Closes #2804 

# Description ℹ️

Adds a 'Resources' dropdown to Console's navigation. Dropdown contains links to the feedback repo and the docs site, both URLs are set by env vars

# Demo 📺

Screenshot 2023-03-21 at 15 22 47

### CODE REVIEW

File: apps/trading/components/navbar/navbar.tsx [hunk 1]
```diff
@@ -75,12 +79,30 @@ export const Navbar = ({
...
-              <span>{t('Governance')}</span>{' '}
-              <Icon name="arrow-top-right" size={3} />
-            </span>
-          </ExternalLink>
+          <NavExternalLink href={tokenLink(TOKEN_GOVERNANCE)}>
+            {t('Governance')}
+          </NavExternalLink>
+        </NavigationItem>
+        <NavigationItem>
+          <NavigationTrigger>{t('Resources')}</NavigationTrigger>
```
Comment (macqbat) [inline apps/trading/components/navbar/navbar.tsx:87]: question (non-blocking): if no VEGA_DOCS_URL nor GITHUB_FEEDBACK are given, Resources dropdown will be empty?

**Review by macqbat — APPROVED** [review]

**Review by daro-maj — COMMENTED** [review]
What about e2e tests ? Let me know if you need any help.

File: apps/trading/components/navbar/navbar.tsx (hunk 1 above)
Comment (mattrussell36) [reply apps/trading/components/navbar/navbar.tsx]: Done

**Comment by mattrussell36** [comment]
> What about e2e tests ? Let me know if you need any help.

@daro-maj added some in the last commit. Will merge but please take a look make sure you are happy with coverage

**Pull request merged**